apps/ingest-from-bucket/README.md

```diff
@@ -116,6 +116,39 @@ docker run \
 
 See [Common Parameters](../../README.md#common-parameters) for additional options.
 
+## Permissions
+
+In S3, the minimal permissions for running the workflow are as follows:
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "VisualEditor0",
```

Comment:
I would drop the `Sid` field. It's optional and doesn't add anything, at least with these values,

Reply:
done